src/checkout-buttons/create-checkout-button-initializer.ts

```diff
@@ -35,15 +35,15 @@ import createCheckoutButtonRegistry from './create-checkout-button-registry';
 export default function createCheckoutButtonInitializer(
     options?: CheckoutButtonInitializerOptions
 ): CheckoutButtonInitializer {
-    const host = options && options.host;
+    const { host, locale = 'en' } = { ...options ?? {} };
```

Comment:
const { host, locale = 'en' } =  options ?? {};
```suggestion
    const { host, locale = 'en' } = { ...options ?? {} };
```